definitions/private-law/json/CaseEvent/ServiceOfApplication/CaseEvent-nonprod.json

```diff
@@ -15,6 +15,40 @@
     "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/service-of-application/submitted",
     "EndButtonLabel": "Save and continue",
     "Publish": "True"
+  },
+  {
+    "LiveFrom": "31/10/2022",
+    "CaseTypeID": "PRLAPPS",
+    "ID": "confidentialityCheck",
+    "Name": "Confidentiality Check",
+    "Description": "Confidentiality check",
+    "DisplayOrder": 67,
+    "PreConditionState(s)": "*",
+    "PostConditionState": "*",
+    "SecurityClassification": "Public",
+    "ShowSummary": "Y",
+    "ShowEventNotes": "N",
+    "CallBackURLAboutToStartEvent": "${CCD_DEF_COS_URL}/confidentiality-check/about-to-start",
+    "CallBackURLSubmittedEvent": "${CCD_DEF_COS_URL}/confidentiality-check/submitted",
+    "EndButtonLabel": "Save and continue",
+    "Publish": "True"
```

Comment:
Remove event based confidentiality check, we will follow tab based confidentiality check

Reply:
Done